knowledge-base/grid-customize-delete-confirmation-dialog.md

```diff
@@ -6,18 +6,19 @@ page_title: Customize the Delete Confirmation Dialogs
 slug: grid-kb-customize-delete-confirmation-dialog
 position: 
 tags: grid, customize, delete, confirmation, dialog, message
-ticketid: 1553006
+ticketid: 1553006, 1660439, 1625888
 res_type: kb
 ---
 
 ## Environment
```

Comment:
Consider editing the Description section by applying the current standards with "This KB also answers..."